client/css/style.css

```diff
@@ -573,6 +573,7 @@ button {
 }
 #chat .wrap,
 #chat .text a {
+	cursor: pointer;
```

Comment:
Well, what means *necessary*. If you want the links to appear and behave the same as normal html links, then yes. It is necessary. If you are satified with the default behavior, which means the cursor changes to `text`, then it is not necessary.

I like the first option.